django_project/core/settings/project.py

```diff
@@ -13,7 +13,8 @@
 # Project apps
 INSTALLED_APPS += (
     'base',
-    'roles'
+    'roles',
+    'fish',
```

Comment:
Maybe some input from @timlinux, but I would like us to pre-pend some text to identify this as a BIMS module. maybe `bims_module_fish`?

Reply:
No problem to rename it - or we could just maintain that in module the docstrings and keep our names clean and simple.